src/networks.json

```diff
@@ -1172,6 +1172,20 @@
     "start": 1715902,
     "logo": "ipfs://QmTogMDLmDgJjDjUKDHDuc2KVTVDbXf8bXJLFiVe8PRxgo"
   },
+  "53935": {
+    "key": "53935",
+    "name": "DFK Chain",
+    "chainId": 53935,
+    "network": "mainnet",
+    "multicall": "0x5b24224dC16508DAD755756639E420817DD4c99E",
+    "rpc": [
+      "https://subnets.avax.network/defi-kingdoms/dfk-chain/rpc",
+      "https://avax-dfk.gateway.pokt.network/v1/lb/6244818c00b9f0003ad1b619/ext/bc/q2aTwKuyzgs8pynF7UXBZCU7DejbZbZ6EUyHr3JQzYgwNPUPi/rpc"
```

Comment:
we can remove this node , at this point we are only looking to add full archive node, but this doesn't look like so

Reply:
Removed it